.nvmrc

```diff
@@ -0,0 +1 @@
+v6.10.0
```

Comment:
How about leveraging the `engines` on `package.json` instead? I'm thinking probably `node >=4` should do it.